fieldsolver/ldz_magnetic_field.cpp

```diff
@@ -192,6 +192,36 @@ void propagateSysBoundaryMagneticField(
    }
 }
 
+/*! \brief Low-level magnetic field projection function.
+ *
+ * Projects the magnetic field according to the system boundary conditions.
+ *
+ * \param perBGrid fsGrid holding the perturbed B quantities at runge-kutta t=0
+ * \param perBDt2Grid fsGrid holding the perturbed B quantities at runge-kutta t=0.5
+ * \param technicalGrid fsGrid holding technical information (such as boundary types)
+ * \param i,j,k fsGrid cell coordinates for the current cell
+ * \param sysBoundaries System boundary conditions existing
+ * \param RKCase Element in the enum defining the Runge-Kutta method steps
+ *
+ * \sa propagateMagneticFieldSimple propagateMagneticField
+ */
+void projectSysBoundaryMagneticField(
```

Comment:
One thing that really confused me when looking at this diff for the first time is the two different meanings of the word "project" in the source code.

There is Project::setProjectBField, which deals with the magnetic field set by a project.
And now there is projectSysBoundaryMagneticField, which dieals with the magnetic field *projection*.

Should we, to minimize confusion, maybe try to consistently use the form "projection" here? So this identifier would be sysBoundaryMagneticFieldProjection?